android/src/main/java/it/innove/BleManager.java

```diff
@@ -494,8 +494,9 @@ public static WritableArray bytesToWritableArray(byte[] bytes) {
 		return value;
 	}
 
+
 	@Override
-	public void onActivityResult(Activity activity, int requestCode, int resultCode, Intent data) {
+	public void onActivityResult(int requestCode, int resultCode, Intent data) {
```

Comment:
Interesting that this had to change, too. Reason?

Reply:
I opened the project in Android Studio and let the IDE sync the project and it detected this as an improperly overridden method.

https://developer.android.com/training/basics/intents/result.html
https://developer.android.com/reference/android/app/Activity.html#onActivityResult(int, int, android.content.Intent)

This signature matches the examples and documentation.